nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java

```diff
@@ -184,11 +196,17 @@ public void testExportWithParams() throws Exception {
         // Check header
         List<String> header = Arrays.asList(lines.get(0).split(","));
         assertArrayEquals(
-                new String[] { "dc:contributors", "dc:coverage", "dc:created", "dc:creator", "dc:description",
-                        "dc:expired", "dc:format", "dc:issued", "dc:language", "dc:lastContributor", "dc:modified",
-                        "dc:nature", "dc:publisher", "dc:rights", "dc:source", "dc:subjects", "dc:title", "dc:valid",
-                        "cpx:complex/foo" },
-                header.subList(18, 37).toArray());
+                new String[] { "dc:contributors", "dc:coverage", "dc:coverage[label]", "dc:created", "dc:creator",
+                        "dc:description", "dc:expired", "dc:format", "dc:issued", "dc:language", "dc:lastContributor",
+                        "dc:modified", "dc:nature", "dc:nature[label]", "dc:publisher", "dc:rights", "dc:source",
+                        "dc:subjects", "dc:subjects[label]", "dc:title", "dc:valid", "cpx:complex/foo" },
+                header.subList(18, 40).toArray());
```

Comment:
Don't remember why we're skipping 18 elements. Can you add a comment ?

Reply:
Because there are the system properties before, I'll add a comment.